.gitignore

```diff
@@ -23,3 +23,4 @@ TAGS
 .vs
 .vscode
 .idea/
+*~
```

Comment:
I use Emacs and have plenty of files lying around like ./test/common/stats/stats_speed_test.cc.~1~ and they don't cause any trouble. I was wondering if this was some general setting I had, but I can't find it.

So my opinion is that git does the right thing for emacs old-version files by default and this is not necessary.